### ISSUE TITLE
Gui tests are randomly failing

### ISSUE THREAD

**Issue by rainman110** [dlr-gtlab/gtlab-core#1113]
I suppose, there is a timing issue. I need to figure out, which tests are normally failing.

**Comment by mariusalexander** [comment]
Is this still the case?

**Comment by rainman110** [comment]
> Is this still the case?

Yes, it happens from time to time. It then help to reexecute the tests a couple of times.

**Comment by rainman110** [comment]
@mariusalexander Here's an example of a failing test:

[All Suites](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html) > [gtlab_core_gui_tests](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html#test=-0) > [tst__init__](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html#test=-0-0) > [Initialize and setup GTlab for GUI Testing](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html#test=-0-0-0) > [Load the standard project](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html#test=-0-0-0-0) > [And the standard project is removed from the session](https://gtlab-coredev.pages.gitlab.dlr.de/-/github-mirrors/gtlab-core-mirror/-/jobs/3448031/artifacts/gui_tests_web/index.html#test=-0-0-0-0-3)

**Comment by jensschmeink** [comment]
As there is more stability after the last updates this issue will be closed and reopened on demand
